src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlCommand.cs

```diff
@@ -506,7 +506,7 @@ private SqlInternalConnectionTds InternalTdsConnection
             }
         }
 
-        private static bool IsRetryEnabled => LocalAppContextSwitches.IsRetryEnabled;
+        private bool IsRetriable => SqlConfigurableRetryFactory.IsRetriable(RetryLogicProvider);
```

Comment:
I prefer the name `IsRetryEnabled` over `IsRetriable`. The name `IsRetryEnabled` indicates the state of a configuration option "I have enabled this named thing" but the name `IsRetriable` feels more like it is trying to express a property of the contents "this thing has an aspect which makes it have the named property/behaviour". 

`IsRetriable` feels like `string.IsNullOrEmpty` where you are finding a piece of information about the object.

Reply:
If `IsRetriable` is the most accurate description it might be better to make it `CanRetry`

Reply:
My thought on this is that `IsRetriable` is describing a feature of the retry logic provider, so I also agree that it feels a little off in the context it is used.

Maybe `IsProviderRetriable` or `IsProviderRetryEnabled` would be better?